tidy3d/plugins/invdes/optimizer.py

```diff
@@ -130,7 +136,7 @@ def continue_run(
             post_process_val = aux_data["post_process_val"]
 
             # update optimizer and parameters
-            params, opt_state = self.update(parameters=params, state=opt_state, gradient=-grad)
+            params, opt_state = self.update(parameters=params, state=opt_state, gradient=grad)
```

Comment:
hm, if 
```
direction_multiplier = 1 if maximize else -1
obj_fn_val = direction_multiplier * post_process_val - penalty_value
```
then shouldn't we be using `-grad`  here since we're always trying to maximize this quantity?

Reply:
oops

Reply:
not sure it's pushed? just fyi